src/reputation/related_models/bounty.py

```diff
@@ -37,17 +37,23 @@ class Type(models.TextChoices):
     CANCELLED = "CANCELLED"
     EXPIRED = "EXPIRED"
     CLOSED = "CLOSED"
+    REVIEW_PERIOD = "REVIEW_PERIOD"
     status_choices = (
         (OPEN, OPEN),
         (CLOSED, CLOSED),
         (CANCELLED, CANCELLED),
         (EXPIRED, EXPIRED),
+        (REVIEW_PERIOD, REVIEW_PERIOD),
     )
 
     expiration_date = models.DateTimeField(
         null=True,
         default=get_default_expiration_date,  # Can be null for author claim bounties
     )
+    review_period_days = models.IntegerField(
```

Comment:
Rather than storing the review period time in days for each and every bounty, let's make it configurable via a constant somewhere such as `settings.py`. Unless I'm missing something I think it would be best to store the date of the review period in the DB so it is more clear when it ends